deploy.sh

```diff
@@ -28,7 +28,6 @@ push_helm_charts() {
 	mkdir -p "$CHARTDIR/csi-charts/docs/$PACKAGE"
 	cp -R "./charts/ceph-csi-$PACKAGE" "$CHARTDIR/csi-charts/docs/$PACKAGE"
 	pushd "$CHARTDIR/csi-charts/docs/$PACKAGE" >/dev/null
-	helm init --client-only
```

Comment:
Just wondering is there a replacement for above option in helm3

Reply:
please check https://github.com/helm/helm/issues/6996#issuecomment-555010868